configuration/items.md

```diff
@@ -525,18 +529,20 @@ See the [Hue Emulation]({{base}}/addons/io/hueemulation/readme.html) or [HomeKit
 ### Binding Configuration
 
 One of the greatest strengths of an openHAB automation system is the sheer number of devices you can interact with.
-See "[currently available Bindings]({{base}}/addons/bindings.html)" for a list of available bindings.
-This capability of interacting with real-world things is enabled through the association of bindings with Items.
-Once a binding is associated with an Item, the state of a thing is reflected in various openHAB UIs (e.g., you can see if a light is ON or OFF).
-Additionally, you have the opportunity to interact with that thing through its Item, if interaction is supported by the binding (e.g., you can command the light to turn ON or turn OFF).
+See "[currently available Bindings]({{base}}/addons/bindings.html)" for a list of available Bindings.
+This capability of interacting with real-world things is enabled through the association of Bindings with Items.
+
+Once an Item is associated with a Binding, the state of one aspect of a device is reflected in openHAB (e.g., you can see if a light is on or off in one of the user interfaces).
+Additionally, you have the opportunity to interact with a device through its Items, if interaction is supported for that aspect of the device (e.g., you can command the light to turn ON or turn OFF).
```

Comment:
I've changed a few sentences here because the concept of things and the relation between Items, Bindings and aspects of a device (channels/properties/capabilities/datapoints) was a bit mixed - both from a general view and wrt the difference between 1.x and 2.x bindings.